app/extlinux_control.py

```diff
@@ -716,12 +716,16 @@ def get_version(self) -> str:
 
     def boot_ctrl_pre_update(self, version):
         logger.debug("entering pre-update...")
+        self.store_slot_in_use_file(
```

Comment:
@Bodong-Yang is this required? fixing bug?

Reply:
I think this is a redundant line, I think we can remove it.
Because I notice there is a same line below, write the current slot's `slot_in_use` file.